packages/apps-config/src/api/constants.ts

```diff
@@ -23,6 +23,8 @@ export const POLKADOT_DENOM_BLOCK = new BN(1248328);
 
 export const ROCOCO_GENESIS = getGenesis('rococo');
 
+export const PASEO_GENESIS = getGenesis('paseo');
```

Comment:
```suggestion
export const PASEO_GENESIS = '0x74300973617e2936e22d46e94fee5016a1a514747ae108277b770d02b47d37d9';
```

PS: We actually had RoccorWestend the same way for a very long time. (This way is certainly more "agile", it takes quite a bit of time for common-repo changes to make the way in - so it moves quite a bit faster)